app/controllers/galleries_controller.rb

```diff
@@ -1,5 +1,7 @@
 class GalleriesController < ApplicationController
   before_action :require_login, only: [:new, :create, :destroy, :edit, :update]
+  before_action :correct_user, only: [:destroy, :edit, :update]
```

Comment:
Again, I'd change the name of this filter to something a bit more expressive, e.g. `user_can_modify`